warehouse/locale/de/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,41 @@
+# German translations for PROJECT.
+# Copyright (C) 2019 PyPA
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2019-08-27 12:01-0400\n"
+"PO-Revision-Date: 2019-08-26 18:02-0400\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: de\n"
+"Language-Team: de <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.7.0\n"
+
+#: warehouse/accounts/views.py:59
+msgid "There have been too many unsuccessful login attempts. Try again later."
```

Comment:
Just a quick note, feel free to disregard.

In the past I've been bitten by having the `msgid` being the actual english copy. While more intuitive when reading it in the code it made simple english copy changes much harder.

Is it worth considering a coded `msgid` + an american english de facto translation?

Reply:
I'd lean towards using IDs for the reasons that @yeraydiazdiaz has pointed out.

Reply:
Decision has been made to use the English copy as the msgid, Conversations in https://github.com/pypa/warehouse/pull/6598 supported this and ultimately it seems that as far as translator experience goes it should serve us best.